README.md

```
@@ -1 +1,226 @@
+
+### 기타 명령어
+
+```bash
+./gradlew build     # 프로젝트 빌드
+./gradlew test      # 테스트 실행
+./gradlew clean     # 빌드 파일 정리
+```
+
+## 프로젝트 구조
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)